src/main/resources/logsForLocalDev.json

```diff
@@ -631,7 +632,7 @@
       "project_id": "teammates-john",
       "module_id": "default",
       "zone": "asia-east2-3",
-      "version_id": "7-0-0"
+      "version_id": "8-0-0"
     },
     "severity": "INFO",
     "logName": "projects/teammates-john/logs/stdout",
```

Comment:
I think the `X-WEB-VERSION` should also be added in the `requestHeaders`, since it is now part of the `requestHeaders`.